### DISCUSSION TITLE
我们常说的开源文化，到底是什么？ - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#38] [Announcements]
# 我们常说的开源文化，到底是什么？ - 「开源之道」

https://opensourceway.community/posts/opensource_culture/what-is-open-source-culture-when-people-said/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)


> ...ApacheCon Asia 2021和 2022 两年均设置了文化的专题，2021颗粒无收

其实, 作者已经分析清楚原因了:

> ...文化并没有那么多神秘，其实就是我们的日常

是的, 文化就是生活, 开源文化, 就是将开源作为日常生活那群人的日常;
将开源融入生活后, 变成了自己生存风格, 形成了全新世界观, 析出小众价值观, 进而感染人生观,
日常在各种社会接触面上, 就形成与众不同的亚文化气质;

那么, 在中国根本没有几个能以开源为完全生活状态的人,
自然也就无法吸引/传播/互佐/...出其它新开源者,

文化, 既然是一种生活状态, 那么, 必须有足够数量影响足够面积, 才能被感知成一种新文化;
真正进行开源之事的人, 还没看话剧的人多, 怎么可能有对应专题来分享;

当然, 需要的话, 俺可以贡献一个, 之前在 GDG 年会中没通过的议题:

> 如何在开源生活空间中保持心理健康?
